generators/entity-client/templates/react/src/main/webapp/app/entities/entity-dialog.tsx.ejs

```diff
@@ -515,7 +515,7 @@ _%>
 
 const mapStateToProps = storeState => ({
   <%_ otherEntityActions.forEach(val => { _%>
-  <%= val.instance %>: storeState.<%= val.reducer %>.<%= val.instance %>,
+  <%= val.instance %>: storeState.<%= val.reducer %>.entities,
```

Comment:
I think I used val.instance to handle user entity relations as well, can you check if this work with user entity as well